src/modules/auth/auth.controller.ts

```diff
@@ -71,6 +72,8 @@ export class AuthController {
   ) {
     const { token } = params;
     await this.authService.verifyEmail(token);
-    res.status(httpStatus.NO_CONTENT).send();
+    res.render(
```

Comment:
It's not really consistent to return different types of responses in the same controller (.e.g html and json).
I would stick to json and render this view on front end after API request succeeds.